### DISCUSSION TITLE
Portable Charging System for Electric Vehicles (Project 216)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#47] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 216](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Portable%20Charging%20System%20for%20Electric%20Vehicles)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by mouriarty16** [comment]
Hi, I am Harshit. I am a Mechanical engineering student at MANIT, India. I am interested in working and collaborating on this project.

**Reply by Raktacharitra** [reply]
Hi, I am a student at NIT - B and interested in working and collaborating on this project.

**Comment by Raktacharitra** [comment]
Hi, I am a student at NIT-B and interested in working and collaborating on this project.

**Comment by AmanRajSrivastava** [comment]
Hi Everyone! I am an undergrad and I hope to create a good final year project out of project 216. Happy to collaborate.

**Comment by thantgin** [comment]
Hello, my name is Thomas and I am a student at Northeastern University. I am currently working with a team to do this project. We are looking for help in some areas and would love to collaborate.

**Reply by robertogl** [reply]
Hello Thomas,
As you can see other students have been working on this project and are open to collaborate. I am sure they will reply soon.
In the meantime, feel free to ask your questions here, we will try our best to answer quickly.

**Comment by AmanRajSrivastava** [comment]
Has anybody considered using ultracapacitors for secondary storage system ? please shed some lights on why (not) to use it.

**Reply by drschandel** [reply]
Hi, that's a good question, Aman. 

Personally I think ultracapacitors are good for applications where a lot of energy has to be stored or discharged very quickly, analogous to a mechanical flywheel. But they are generally not appropriate for long-term energy storage. However they can be included as a part of the system to smooth out the energy flow and protect from any current spikes/sudden power draws.

Hope that helps - Dhruv

**Reply by AmanRajSrivastava** [reply]
Sure sir I will read more about using ultracapacitors effectively. I was thinking for different level (2,4) of charging for EV and for level 4 DC -DC charging Ultracapacitors seemed best but there is a tradeoff of cost and portability.

**Comment by Raktacharitra** [comment]
I have done a basic literature review. You may go through it, although I
don't have much experience with it.
I believe I have reviewed the ultracapacitor method somewhere in it.

**Comment by AmanRajSrivastava** [comment]
can anyone share some article/videos/simulation for active cell  balancing? all I can find is mostly on passive cell balancing using resistance switching. Also has anyone tried the code for passive cell balancing ? It seems quite odd to me as the models on Mathworks central use brute force and end up with lowest possible SoC (when i/p are  75, 78, 35,55   the balanced Soc comes out to be 28 for all cells)

**Reply by robertogl** [reply]
Hi @AmanRajSrivastava,

Unfortunately we do not have examples on how to use our tools for active cell balancing. However, Just by doing a simple Google search I found [this article](https://www.cselectricalandelectronics.com/active-cell-balancing-using-a-flyback-converter-simulation-in-matlab-simulink/) that could be of interest. The article adopts a flyback converter block, which you can find [here](https://www.mathworks.com/help/physmod/sps/ug/flyback-converter.html) and is part of Simscape Electrical. 
If you do a specific search on IEEE Xplore you can find many papers, however, that can be useful only if you a subscription to the IEEE database.

**Comment by lipun7naik** [comment]
"Hi, I am a student at Veer Surendra Sai University of Technology Burla and interested in working and collaborating on this project 216".

**Reply by Ayushalt17** [reply]
Hi I was a student too of the same college. Is there any way we could interact as to discuss the models

**Reply by lipun7naik** [reply]
https://www.linkedin.com/in/lipun-kumar-naik-78356b20b/
We can interact via linkedin. Let's get connected.

**Comment by Ayushalt17** [comment]
Can anyone tell me what are the perks of completion of this project ?

**Reply by robertogl** [reply]
Hi @Ayushalt17,

The rewards are: 
- An official MathWorks certificate of accomplishment
- LinkedIn recommendation
- Referral for positions at MathWorks (depending on the quality of the submission)
- MathWorks t-shirt.

I hope these are good incentives to complete and share your project!

**Comment by amrmarey15** [comment]
Hello, my name is Amr. I am a student at the University of Alberta. I am interested in working and collaborating on this project.

**Comment by hkhan97** [comment]
Hi, my name is Hamza Khan. I am a senior at Texas A&M University in College Station, Texas. I will be working on this project with my team and mentor throughout Summer '22.

**Reply by robertogl** [reply]
Hi @hkhan97,

Great to know you will be working on this project! Please do not hesitate to ask any question regarding this project here.

**Comment by ThanmayaSri** [comment]
Hi, I am a Thanmaya Sri , just done with my Bachelors from VNR college. I'm interested in working and collaborating on this project

**Comment by harine17** [comment]
Hi I am Harine, just graduated from Amrita School of Engineering, Bangalore. I am interested in working and collaborating in this project

**Comment by engrhameedahmed** [comment]
Hi there I am Hameed Ahmed, an Electrical Engineer from Pakistan. I am interested in working and collaborating on this project as it is one of the best concepts for modern vehicles. If anyone who is working on this project and serious about this project do comment so we could collaborate on this project.

**Reply by SoufianeLaebouss** [reply]
Hi, I am Soufiane Laebouss, I study Electrical Engineer in France. If you want we could work together on this project.

**Comment by Agr-sagar** [comment]
Hi @robertogl sir,

I'm exploring the idea of forming a multinational student team to collaborate on a project focused on designing a portable charger for electric vehicles. This project is driven by the global trend towards electrification and the need for innovative solutions in the field.

I believe that a diverse team of students from different universities and countries can bring valuable insights and skills to this project. It could be a great opportunity for international collaboration and knowledge exchange.

Please let me know whether I can collaborate and submit the same project with different universities and nations or not.

**Reply by robertogl** [reply]
Hello @Agr-sagar,

Yes, you can collaborate with students from other universities and countries! I like the idea and I am fully supportive.

**Comment by saadjawad79** [comment]
Hi there I am Muhammad Saad, an Electrical Engineering undergraduate from NUST, Pakistan. I am interested in working and collaborating on this project.

**Comment by Santunu123** [comment]
Hi sir @robertogl ,i am santunu kumar barik,a 3rd year b.tech undergraduate from vssut,burla under electrical engineering depaartment.I am eager to work in this project.
